packages/mobile/src/localization/localization.ts

```diff
@@ -1 +1 @@
-export { i18 } from './i18n';
+export { i18, resources } from './i18n';
```

Comment:
```suggestion
export { i18 } from './i18n';
```

Reply:
can't change, issues will hints won't work, and useTranslation get recursion depth issue